docs/src/modules/utils/parseMarkdown.js

```diff
@@ -37,7 +37,7 @@ export const demoRegexp = /^"demo": "(.*)"/;
 export function getContents(markdown) {
   return markdown
     .replace(headerRegExp, '') // Remove header information
-    .split(/^{{|}}$/gm) // Split markdown into an array, separating demos
+    .split(/^{{("demo":[^}]*)}}$/gm) // Split markdown into an array, separating demos
```

Comment:
Fix for https://5d14e1b386595e00096c0e9a--material-ui.netlify.com/components/text-fields/#integration-with-3rd-party-input-libraries